ClientApp/src/NavBar/NavBar.scss

```diff
@@ -80,6 +80,7 @@ li a:hover {
   right: 0;
   color: $light-text;
   z-index: 1001;
+  background-color: transparent !important;
```

Comment:
another !important we probably want to get rid of :)

Reply:
this one is actually addressed in another pull request but I can delete it here too :)